app/components/async-list/template.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="empty">
     <LoadingIcon />
   </div>
-{{else if @data.last.isError}}
+{{else if (not @data)}}
```

Comment:
`@data.isError` is a official [task state](https://ember-concurrency.com/api/TaskInstance.html) which is also available in here. It's also mentioned [here](https://ember-resources.pages.dev/funcs/util_ember_concurrency.trackedTask):

> (and all other properties on a [TaskInstance](https://ember-concurrency.com/api/TaskInstance.html))